test/Tool/range6.opt

```diff
@@ -0,0 +1,18 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: %FileCheck %s < %t
+
+; CHECK: LGTM
+
+; min = 1024 - 2048 [lower_1 - upper_2]
+; max = 4096 - (-1024) [upper_1 - lower2]
+
+%0:i32 = var (range=[1024,4097))
+%1:i32 = var (range=[-1024,2049))
+%2:i32 = subnw %0, %1
+%3:i1 = sle -1024, %2
+%4:i1 = slt %2, 5120
+%6:i1 = and %3, %4
+cand %6 1:i1
+
```

Comment:
I think you can delete one of range5 and range6 since they seem to test the same thing